packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -41,15 +41,19 @@
       </div>
       <div class="flex items-center gap-x-3">
         <div v-if="workspaceInfo.billing" class="flex-1 md:flex-auto">
-          <WorkspacePageVersionCount
-            :versions-count="workspaceInfo.billing.versionsCount"
-          />
+          <button class="block" @click="openSettingsDialog('billing')">
```

Comment:
We should probably create a const for the menu item keys and use that (I also didnt do that for general), maybe something like 

```
const SettingMenuKeys = Object.freeze(<const>{
  Workspace: {
    General: 'workspaceGeneral',
    Billing: 'workspaceBilling'
    // etc
  },
  Server: {
    General: 'serverGeneral'
    // etc
  },
  User: {
    Details: 'userDetails'
    // etc
  },
})
```

And update the composable (`lib/settings/composables/menu.ts`) to use these keys 

```
  const workspaceMenuItems = shallowRef<SettingsMenuItems>({
    [SettingMenuKeys.Workspace.General]: {
      title: 'General',
      component: SettingsWorkspaceGeneral,
      permission: [Roles.Workspace.Admin, Roles.Workspace.Member, Roles.Workspace.Guest]
    },
   // etc
  })
  ```
  
  Then here we can do `@click="openSettingsDialog(SettingMenuKeys.Workspace.General)"` to assure it's an actual existing item (and avoid typos)